synapse/config/oidc_config.py

```diff
@@ -158,6 +159,10 @@ def generate_config_section(self, config_dir_path, server_name, **kwargs):
           #
           #skip_verification: true
 
+          # Uncomment to add oidc token to the account instead of failing when user already exists. Defaults to false.
```

Comment:
I don't think this comment is very descriptive -- what's the token that's being added? Why would I turn this on?

```suggestion
          # Uncomment to allow a user logging in via OIDC to match a pre-existing account instead
          # of failing. This could be used if switching from password logins to OIDC. Defaults to false.
```